ui/mods/msu/utilities.js

```diff
@@ -1,6 +1,7 @@
 var MSU = {
 	ID : "mod_msu",
 	Hooks : {},
+	Popup : Screens.MSUPopup
```

Comment:
What is this for?

Reply:
It's just easier to remember to use MSU.Popup over Screens.MSUPopup.